buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -291,8 +291,8 @@ class VagrantTestPlugin implements Plugin<Project> {
                     throw new StopExecutionException("upgrade version is before 6.3.0")
                 }
             }
-            file "${archivesDir}/upgrade_is_oss"
-            contents ''
+            file = "${archivesDir}/upgrade_is_oss" as File
```

Comment:
The equals is ok, other parts of the build use it as well and I think it's the direction Gradle is going into w.r.t. task properties.
We could do without the `as File` if we have a `setFile(String file)` method, see my previous suggestion.